sources/logdog/src/log_request.rs

```diff
@@ -0,0 +1,69 @@
+//! Provides the list of commands that `logdog` will run.
+
+/// A command the filename for its output.
+pub(crate) struct LogRequest<'a> {
+    pub(crate) filename: &'a str,
+    pub(crate) command: &'a str,
+}
+
+/// Returns the standard list of `logdog` commands.
+pub(crate) fn log_requests<'a>() -> &'a [LogRequest<'a>] {
+    &[
+        (LogRequest {
+            filename: "os-release",
+            command: "cat /etc/os-release",
+        }),
```

Comment:
The expansion of this file because of the struct is unfortunate.  I think the best options are to still use tuples, but map them to LogRequests, e.g.

```
&[
   (filename, command),
   (filename, command),
].map(|(filename, command)| LogRequest { filename, command })
```

...and then return something like `impl Iterator<Item = LogRequest>`...

Or to have this module that users see just include the tuples, and map them to LogRequests where we call this function.  What do you think?